01_fib/ailyntang/fib.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func fib(n int64) {
```

Comment:
the type of `n` should be just `int` - there is no need for a 64-bit integer here as n has to be between 1..92 anyway. Only use the specific size `int` types where it matters - such as where you have when making the slice - that is correct because you need an `int64` to hold fib(92).

See https://yourbasic.org/golang/int-vs-int64/ for a little more detail.

Reply:
That makes a lot of sense, thanks for the explanation.